Makefile

```diff
@@ -49,3 +56,19 @@ dev:
 	@echo "Running with local development themes"
 	docker run --env-file .env -it --rm -v "$(PWD)/render:${WORKDIR}/render" -v "$(PWD)/templates:${WORKDIR}/templates" ${ORG}/${NAME} ${FLAGS}
 	python -m webbrowser -t "file://$(PWD)/render/index.html"
+
+# Running locally 
+# We optionally include our .env file if it exists (compatibility with the docker
+# --env-file) and then create a vitualenv as needed for running our local command.
+ifneq (,$(wildcard ./.env))
+    include .env
```

Comment:
I believe this is no longer necessary now that #18 is merged.

Reply:
Seems to work. Fixed in 58fcd5e0573d075ea6d5eb57798ec2da9fe0a211